pkg/execution/svccatadmission/internaldns.go

```diff
@@ -0,0 +1,132 @@
+package svccatadmission
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/golang/glog"
```

Comment:
Use zap for logging and please don't use the global logger, inject one instead like other components do.

Reply:
This line was pushed by mistake. Will be fixed on next commits.

Reply:
Using Sugarred log from context now

Reply:
fixed